src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: 'Quotas & Events'
+sidebar_order: 0
+---
+
+Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events to be tracked. This number of events is your quota. When an event occurs, it counts toward your quota.
+
+Sentry’s flexibility means you can exercise fine-grained control over which events count toward your quota.
+
+## Key Terms
+
+Let’s clarify a few terms to start:
+-   Event - an event is one instance of you sending Sentry data. Generally, this data is an error. Every event has a unique set of characteristics, called its fingerprint.
+-   Issue - an issue is a grouping of similar events, who all share the same fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
+-   Quota - your quota is the monthly number of events you pay Sentry to track.
+
+## What Counts Toward My Quota, an Overview
+
+Sentry completes a thorough evaluation of each event to determine if it counts toward your quota, as outlined in this overview. Detailed documentation for each evaluation is linked throughout. Before completing any of these evaluations, Sentry confirms that each event includes a valid DSN and project as well as whether the event can be parsed and contains valid fingerprint information; if any of these items are missing or incorrect, the event is rejected.
+
+1. **SDK configuration**
+
+      The SDK configuration either allows the event or filters the event out. For more information, see [Outbound Filters in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md -%}#outbound-filters) or [Filtering Events]({%- link _documentation/error-reporting/configuration/index.md -%}).
+
+2. **SDK sample rate**
+
+      If a sample rate is defined for the SDK, Sentry evaluates whether this event should be sent as a representative fraction of the sampled events. Setting a sample rate is documented for each SDK. For more information, see [Configuration, sampleRate]({%- link _documentation/error-reporting/configuration/index.md -%}#sample-rate).
+
+3. **Quota availability**
+
+      Events that exceed your quota are not sent. To add to your quota or review what happens when you exceed it, see [Increasing Quotas]({%- link _documentation/accounts/quotas/index.md -%}#increasing-quotas).
+
+4. **Event repetition**
+  -   If you have intervened to discard events with the same fingerprint, the event does not count toward your quota. For more information, see [Delete & Discard in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md  -%}#delete--discard) or [Inbound Filters]({%- link _documentation/accounts/quotas/index.md -%}#inbound-data-filters).
```

Comment:
> to discard events with the same fingerprint

Can we clarify this by literally mentioning _"Delete and Discard"_? I only realized what this means when reading the hash tag in the link.

Also, "Inbound filters" are quite a different concept and mentioned explicitly below -- might it be confusing to mention it here?

Reply:
Good catch. Will edit to: If you have intervened to Delete and Discard events with the same fingerprint, the event does not count toward your quota.

Reply:
@PeloWriter 
Also, you discard an Issue not an event. maybe write "if you intervened to discard an Issue, then future events with the same fingerprint will not be accepted/counted towards your quota"

Also be careful here, because a really common question is, "If I Delete&Discard an issue, does that mean I won't get billed for the 10,000 events already accepted?" The answer is, "No. those still count as they were already accepted. Everything after 10,000 is what will be rejected (and not billed for)"

so make sure we're not implying that the historical events (already accepted) are not getting billed for.

Reply:
Thanks. Updated to discuss *future* events more clearly here and in the table.